app/views/providers/link_application/copies/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= form_with(
+      model: @form,
+      url: providers_legal_aid_application_link_application_copy_path,
+      method: :patch,
+      local: true,
+    ) do |form| %>
+
+  <%= page_template(
+        page_title: t(".question", lead_application_reference: @lead_application_reference),
+        form:,
+        template: :basic,
+      ) do %>
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-1"><%= t(".heading") %></h1>
+    <%= govuk_summary_list(actions: false, classes: " govuk-summary-list--no-border govuk-!-margin-bottom-20") do |summary_list|
```

Comment:
Is there such thing as a `govuk-!-margin-bottom-20`? I thought it only went up to 9?

Reply:
Weird - I know I was having spacing issues. I think this was _supposed_ to be a 2 bit I added 0.
A bit of playing now and it's not needed at all now! 🎉